assets/js/components/community/overrides/SubcommunityCreateForm.js

```diff
@@ -0,0 +1,301 @@
+import { i18next } from "@translations/invenio_communities/i18next";
+import { Formik } from "formik";
+import _get from "lodash/get";
+import _isEmpty from "lodash/isEmpty";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import {
+  FieldLabel,
+  RadioField,
+  SelectField,
+  TextField,
+  withCancel,
+  http,
+} from "react-invenio-forms";
+import SearchDropdown from "../SearchDropdown";
+import { Button, Divider, Form, Grid, Header, Icon, Message } from "semantic-ui-react";
+import { CommunityApi } from "@js/invenio_communities/api";
+import { communityErrorSerializer } from "@js/invenio_communities/api/serializers";
+
+class SubcommunityCreateForm extends Component {
+  state = {
+    error: "",
+    hasCommunity: false,
+    communities: [{ value: "Loading..." }],
```

Comment:
question: shouldn't this be computed on rendering? e.g. using a `loading` state

Reply:
we discussed IRL: to be tackled as part of UI improvements. In this case, we will use `defaultValue`of the `SelectField` component